index.js

```diff
@@ -1,310 +1,333 @@
-            "label": "I'm a label"
+            id: "1",
+            label: "I'm a label",
           },
           {
-            "id": "2",
-            "label": "I'm a label",
-            "sub_label": "I'm a sub_label"
-          }
-        ]
+            id: "2",
+            label: "I'm a label",
+            sub_label: "I'm a sub_label",
+          },
+        ],
       },
       {
-        "name": "I'm a date",
-        "type": "date",
-        "id": "date",
-        "is_required": false,
-        "placeholder": "[placeholder]"
+        name: "I'm a date",
+        type: "date",
+        id: "date",
+        is_required: false,
+        placeholder: "[placeholder]",
       },
       {
-        "name": "I'm a datetime",
-        "type": "datetime",
-        "id": "datetime",
-        "is_required": false,
-        "placeholder": "[placeholder]"
+        name: "I'm a datetime",
+        type: "datetime",
+        id: "datetime",
+        is_required: false,
+        placeholder: "[placeholder]",
       },
       {
-        "name": "I'm a typeahead",
-        "type": "typeahead",
-        "id": "typeahead_half_width",
-        "is_required": false,
-        "typeahead_url": "https://localhost:8000/search/typeahead",
-        "placeholder": "[half width]",
-        "width": "half"
+        name: "I'm a typeahead",
+        type: "typeahead",
+        id: "typeahead_half_width",
+        is_required: false,
+        typeahead_url: "https://localhost:8000/search/typeahead",
+        placeholder: "[half width]",
+        width: "half",
       },
       {
-        "name": "I'm a typeahead",
-        "type": "typeahead",
-        "id": "typeahead_full_width",
-        "is_required": false,
-        "typeahead_url": "https://localhost:8000/search/typeahead",
-        "placeholder": "[full width]",
-        "width": "full"
-      }
+        name: "I'm a typeahead",
+        type: "typeahead",
+        id: "typeahead_full_width",
+        is_required: false,
+        typeahead_url: "https://localhost:8000/search/typeahead",
+        placeholder: "[full width]",
+        width: "full",
+      },
     ],
-    "on_change_callback": "https://localhost:8000/form/onchange"
-  }
-}
+    on_change_callback: "https://localhost:8000/form/onchange",
+  },
+};
 
 typeahead_response = {
-  "items": [
+  items: [
     {
-      "title": "I'm a title",
-      "subtitle": "I'm a subtitle",
-      "value": "some_value",
-      "icon_url": "https://placekitten.com/16/16",
+      title: "I'm a title",
+      subtitle: "I'm a subtitle",
+      value: "some_value",
+      icon_url: "https://placekitten.com/16/16",
     },
     {
-      "title": "I'm a title",
-      "subtitle": "I'm a subtitle",
-      "value": "some_value",
-      "icon_url": "https://placekitten.com/16/16",
-    }
-  ]
-}
+      title: "I'm a title",
+      subtitle: "I'm a subtitle",
+      value: "some_value",
+      icon_url: "https://placekitten.com/16/16",
+    },
+  ],
+};
```

Comment:
consider: Might be nice to leave a comment indicating why we don't have an example of App Rules - infrastructure behind App Rules makes it infeasible to have an example of App Rules running as a local server.

Reply:
Will note this in the README